src/ecmech/ECMech_evptnWrap.h

```diff
@@ -400,6 +400,14 @@ namespace ecmech {
                _complete = true;
             };
 
+            // Constant getter functions to return the underlying templated classes.
+            // Uses for these could be for example to compute the sample D^p tensor
+            // using the symmetric schmid tensor from the SlipGeom class.
+            const SlipGeom & getSlipGeom() const { return _slipGeom; }
```

Comment:
Should we add a comment here warning folks that the implementation details of these underlying templated classes could change? That is, we are not promising backward compatibility if people make use of details of these class implementations.

Reply:
We can definitely add that in there, since it doesn't hurt. Although, I would hope that people notice that we're still pre-1.0, so stability of the API still isn't guaranteed.

Reply:
@nrbarton updated the comment to mention this.